dashboard/src/pages/apps/new.tsx

```diff
@@ -8,27 +8,40 @@ import {
   AuthenticationType,
   PortPublicationProtocol,
   CreateApplicationRequest,
+  ApplicationConfig,
+  BuildConfigRuntimeBuildpack,
+  CreateWebsiteRequest,
+  PortPublication,
+  BuildConfigRuntimeCmd,
+  BuildConfigRuntimeDockerfile,
+  BuildConfigStaticCmd,
+  BuildConfigStaticDockerfile,
+  RuntimeConfig,
 } from '/@/api/neoshowcase/protobuf/gateway_pb'
-import { A, useSearchParams } from '@solidjs/router'
+import { A, useNavigate, useSearchParams } from '@solidjs/router'
 import { BsArrowLeftShort } from 'solid-icons/bs'
 import { Container } from '/@/libs/layout'
 import { vars } from '/@/theme'
 import { styled } from '@macaron-css/solid'
 import { Button } from '/@/components/Button'
 import { Checkbox } from '/@/components/Checkbox'
 import { providerToIcon, repositoryURLToProvider } from '/@/libs/application'
+import { createStore, SetStoreFunction } from 'solid-js/store'
+import { Empty } from '@bufbuild/protobuf'
+import toast from 'solid-toast'
+import { ConnectError } from '@bufbuild/connect'
 
```

Comment:
https://github.com/traPtitech/NeoShowcase/pull/542#discussion_r1211679523 についての具体的な対応策です①

まずこのような、classのインスタンスを受け取ってstoreを作成してstoreだけ返す関数を作ってあげて...(続きます)

```suggestion
// ストアを作成するためのユーティリティ関数
const storify = <T extends {},>(class_: T) => {
  const [store, _] = createStore(class_)
  return store
}
```